modules/launcher/src/test/java/org/mule/runtime/module/launcher/service/LifecycleFilterServiceProxyTestCase.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.runtime.module.launcher.service;
+
+import static org.mockito.Mockito.mock;
+import org.mule.runtime.api.service.Service;
+import org.mule.runtime.core.api.lifecycle.Startable;
+import org.mule.runtime.core.api.lifecycle.Stoppable;
+import org.mule.tck.junit4.AbstractMuleTestCase;
+
+import org.junit.Test;
+
+public class LifecycleFilterServiceProxyTestCase extends AbstractMuleTestCase
+{
+
+    @Test(expected = UnsupportedOperationException.class)
```

Comment:
The reason is that another that the expected one may throw a UnsupportedOperationException and your test case would still pass. That's part why UnsupportedOperationException was introduced in JUnit. 

Other tests case that directly call the method that should fail can use @Test(expected = UnsupportedOperationException.class) since it's easier to read.